### PR TITLE
Fix humphd/brackets#146 - Hook up the reload button in thimble to reload the livedev preview

### PR DESCRIPTION
**Pull request by AJDBenner**
This PR is part of [this](https://github.com/sedge/thimble.webmaker.org/pull/21) PR to fix https://github.com/humphd/brackets/issues/146

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -128,12 +128,25 @@ define(function (require, exports, module) {
...
+                for(var i = 12; i < msgObj.params; i++) {
+                    CommandManager.execute(Commands["VIEW_INCREASE_FONT_SIZE"]);
+                }
+            }
         }
         else if (msgObj.commandCategory === "editorCommand") {
             Editor[msgObj.command](msgObj.params);
         }
+        else if (msgObj.commandCategory === "reloadCommand") {
+            // Klevers reload function call will go here
```
Comment (AJDBenner) [inline main.js]: This is a WIP. @K-LV has a patch that will expose the reload function that we can call here.

**Comment by sedge** [comment]
Looks reasonable.

File: main.js (hunk 1 above)
Comment (sedge) [inline main.js]: Shouldn't this be added in before we land?

Comment (AJDBenner) [inline main.js]: I think so, we'll have to wait for that patch before landing this one.

**Comment by AJDBenner** [comment]
This has been updated to include the reload function.

**Comment by gideonthomas** [comment]
r=me

**Pull request merged**